lib/segment/src/vector_storage/dense/simple_dense_vector_storage.rs

```diff
@@ -214,6 +214,10 @@ impl<T: PrimitiveVectorElement> VectorStorage for SimpleDenseVectorStorage<T> {
         self.vectors.len()
     }
 
+    fn available_size_in_bytes(&self) -> usize {
+        self.available_vector_count() * self.vector_dim() * std::mem::size_of::<T>()
```

Comment:
Another `saturating_mul` candidate?

Reply:
I guess it's safe here because size in bytes on disk cannot be larger than `u64::max`

Reply:
Yes, sounds reasonable :+1: